assignments/prototype-refactor.js

```diff
@@ -7,3 +7,163 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+function GameObject(options) {
+    this.createdAt = options.createdAt;
+    this.dimensions = options.dimensions;
+    this.name = options.name;
+  }
+  
+  GameObject.prototype.destroy = function () {
+    return `${this.name} was removed from the game.`;
+  };
+
+  class GameObject {
+      constructor(props) {
+          this.createdAt = props.createdAt;
+          this.dimensions = props.dimensions;
+          this.name = props.name;
+      }
+      GameObject;prototype;destroy = function () {
```

Comment:
once you use the class syntax, you no longer need the `GameObject.prototype`
you can just do `destroy() {..`